subscriber/config/test-kafka.yaml

```diff
@@ -0,0 +1,65 @@
+graphite:
+  address: "localhost:4756"
+
+ares:
+  # A list of namespaces. Each namespace includes a list of ares clusters
+  namespaces:
+    dev01:
+      - dev-ares01
+
+  # A list of ares clusters. Each cluster includes master node address and timeout in seconds.
+  clusters:
+    clusters:
+      dev-ares01:
+        kafkaProducer:
+          brokers: "kafka11:9000,kafka12:9000"
+          retryMax: 5
+          timeoutInSec: 5
+      dev-ares02:
+        kafkaProducer:
+          address: "kafka21:9000,kafka22:9000"
+          retryMax: 5
+          timeoutInSec: 5
+
+# A list of jobs which are defined in config/jobs
+jobs:
+  # A list of namespaces. Each namespace includes a list of jobs.
+  namespaces:
+    job-test:
+      - test1
+    trips:
+      - job1
+
+controller:
+  # enable sync up with aresDB controller
+  enable: false
+  address: "localhost:5436"
+  refreshInterval: 10
+  serviceName: "ares-controller-staging"
+
+zookeeper:
+  server: "localhost:2181"
+
+etcd:
+  # KV environment, zone, and service from which to write/read KV data (placement
+  # and configuration). Leave these as the default values unless you know what
+  # you're doing.
+  env: test
+  zone: local
+  service: ares-subscriber
+  # Directory to store cached etcd data in.
+  cacheDir: /var/lib/areskv
+  # Configuration to identify the etcd hosts this node should connect to.
+  etcdClusters:
+    - zone: local
+      endpoints:
+        - 127.0.0.1:2379
+
+heartbeat:
+  enabled: true
+  # heartbeatTimeout in second
+  timeout: 30
```

Comment:
config for these values are *time.Duration, which is nanoseconds level?

Reply:
be translated into sec in code

Reply:
update config